data/json/items/tool/misc.json

```
@@ -43,6 +43,31 @@
     "color": "yellow",
     "flags": [ "WATCH", "ALARMCLOCK" ]
   },
+  {
+    "type": "TOOL",
+    "id": "eac_butane_can",
```

```suggestion
    "id": "butane_can",
```
It's generally good to keep item IDs nice and simple so people don't have to remember their idiosyncrasies when looking for them to add them to item groups or tilesets or whatever